airflow-core/src/airflow/api_fastapi/common/parameters.py

```diff
@@ -647,22 +647,22 @@ def to_orm(self, select: Select) -> Select:
         asset_ref_subquery = sql_select(DagScheduleAssetReference.dag_id).distinct()
 
         if self.value:
-            # Filter DAGs that have asset-based scheduling
+            # Filter Dags that have asset-based scheduling
             return select.where(DagModel.dag_id.in_(asset_ref_subquery))
 
-        # Filter DAGs that do NOT have asset-based scheduling
+        # Filter Dags that do NOT have asset-based scheduling
```

Comment:
Oh, how was this over-looked before? Are there more of these in the codebase?

Reply:
I'm afraid so 🫠

Reply:
Okay, I can offer making some rounds of cleanup...presuably and best after this PR is merged not to have conflicts...

Reply:
Yep, I also tried to do a few rounds but still notice it from time to time